app/models/df_w2_accessor.rb

```diff
@@ -1,6 +1,8 @@
 class DfW2Accessor < DfXmlAccessor
   SELECTORS = {
     EmployeeSSN: 'EmployeeSSN',
+    EmployeeNm: 'EmployeeNm',
+    EmployerNm: 'EmployerName BusinessNameLine1Txt',
```

Comment:
[DUST] I just saw that `EmployerNm` is duplicate with `EmployerName` that was already added on line 7. Ideally should keep the one that's there and consider changing `EmployeeNm` to `EmployeeName`.